src/pages/logs-page.js

```diff
@@ -212,15 +195,17 @@ export function LogsPage() {
   return (
     <SearchLayout
       title={
-        <div>
-          <h1 className="text-xl font-semibold">
-            Logs
-            {topology && (
-              <span className="text-gray-600">
-                / {topology.name || topology.text}
-              </span>
-            )}
-          </h1>
+        <div className="flex">
```

Comment:
is this div needed? if `flex` styling is needed, could it be part of SearchableDropdown's className?